### ISSUE TITLE
CDL: Remove text being appended to current CDL copyright notice for SPEC (copyrighted) objects

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/sul-embed#1235]
Can we remove the ", Materials may be subject to copyright." line that is appearing at the bottom of the copyright notice for CDL items?  It seems to appear if the object is using the SPEC (copyrighted) APO.   

Example record: https://searchworks.stanford.edu/view/6772719

![image](https://user-images.githubusercontent.com/14897546/104630731-03705d00-5650-11eb-90f8-e7506a0762a9.png)

**Comment by cbeer** [comment]
I'm assuming this item has since been remediated to remove that text from the digital object?

**Comment by andrewjbtw** [comment]
@cbeer Yes, I took out that statement for this item. I can add it back in if that would help testing. Or I could use one of the stage fixtures.
